auv_sim/auv_sim_bringup/launch/inc/start_world.launch.xml

```diff
@@ -2,10 +2,9 @@
 <launch>
   <arg name="gui" default="true" />
   <arg name="debug" default="false" />
-  <arg name="world_name" default="$(find auv_sim_description)/worlds/seabed.world" />
+  <arg name="world_name" default="$(find auv_sim_description)/worlds/pool.world" />
```

Comment:
Under normal circumstances, you want the pool world to be the default.

Reply:
Which pool? :)

The third world I've mentioned below could be answer to this question. We will have 2 worlds, 1 for RoboSub 1 for the other competition?